src/components/checkboxes/index.md.njk

```diff
@@ -59,7 +59,9 @@ You can add hints to checkbox items to provide additional information about the
 
 ### Conditionally revealing content
 
-You can add conditionally revealing content to checkboxes, so users only see content when it’s relevant to them. For example, you could reveal a phone number input only when a user chooses to be contacted by phone.
+You can add conditionally revealing content to checkboxes, so users only see content when it’s relevant to them.
```

Comment:
Conditionally revealed?

Reply:
Yep, I debated changing that too but wanted to keep the changes to a minimum as I am not a content designer… 😬